02/app.js

```diff
@@ -1,2 +1,36 @@
 const x = 5;
-let result = 0;
\ No newline at end of file
+let result = 0;
+
+//for(i=0; i<=x; i++){
+
+ // result = result + i
+    
+  //  console.log(result)
+
+//}
+
+
+while(result <= x){
+    
+  console.log(result)
+  result++;
+  sum = result;
+  alert(result + sum)
+    
+}
```

Comment:
Zwróć uwagę, że w pętli `for` masz zmienną sterującą `i`, która określa ilość operacji. Podobnie musi być w pętli `while` tylko że sama musisz ją zadeklarować poza pętlą i zwiększana wew niej. Wtedy warunek też musi ją wykorzystywać.

Ważne - u Ciebie `result` to teraz zmienna sterująca - raczej jest to mylące, nazwij ją `j` a w `result` sumuj :)

Teraz warunek jest taki: dopóki `result` będzie mniejszy bądź równy `x` czyli dopóki wynik będzie mniejszy bądź równy 5. Natomiast nasz warunek powinien wyglądać inaczej: `dopóki zmienna sterująca będzie mniejsza bądź równia x` (podobnie jak w `for`)

PS. Wynik sprawdzaj poza pętlą, a nie wew. Pamiętaj, aby wykorzystywać poprzednie wartości tj. `result = result + j`